src/hotspot/cpu/ppc/gc/shared/barrierSetAssembler_ppc.cpp

```diff
@@ -342,7 +342,7 @@ int SaveLiveRegisters::iterate_over_register_mask(IterationAction action, int of
         }
       }
     } else if (vm_reg->is_ConditionRegister()) {
-      // NOP. Conditions registers are covered by save_LR_CR
+      // NOP. Conditions registers are covered by save_LR
```

Comment:
This comment is no longer correct. I don't think that we ever save or restore condition registers at this point. So, I think we can replace this comment by `ShouldNotReachHere(); // live condition registers are unexpected at this point`.

Reply:
Thanks @TheRealMDoerr . I have changed the comment